src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs

```diff
@@ -451,35 +450,36 @@ private void Compact(long removalSizeTarget, Func<CacheEntry, long> computeEntry
             {
                 RemoveEntry(entry);
             }
-        }
 
-        /// Policy:
-        /// 1. Least recently used objects.
-        /// ?. Items with the soonest absolute expiration.
-        /// ?. Items with the soonest sliding expiration.
-        /// ?. Larger objects - estimated by object graph size, inaccurate.
-        private void ExpirePriorityBucket(ref long removedSize, long removalSizeTarget, Func<CacheEntry, long> computeEntrySize, List<CacheEntry> entriesToRemove, List<CacheEntry> priorityEntries)
-        {
-            // Do we meet our quota by just removing expired entries?
-            if (removalSizeTarget <= removedSize)
+            // Policy:
```

Comment:
@maryamariyan, @eerhardt, could you help review the changes to extensions?  Thanks!

Reply:
on it.